### DISCUSSION TITLE
Nginx location, rewrite, upstream and proxy_pass issue, not able to access URL

### DISCUSSION THREAD

**Discussion by gg-gg-v1** [nginx-proxy/nginx-proxy#1695] [Q&A]
I want to access this url -> **201.12.12.12:8080/mystring** via **nginx**.

Below is my configurations:
```
location ^/develop/api/ {
  rewrite ^/develop/api/(.*) /$1  break;
  proxy_pass http://api/$uri$is_args$args;
}

upstream api {
      least_conn;
      keepalive 32;
      zone upstream-counting 64k;
      server 201.12.12.12:8080 max_fails=3 fail_timeout=60 weight=1;
}
```
so for this I am trying to use **201.12.12.12/develop/api/mystring** to point it to **201.12.12.12:8080/mystring** through upstream 'api' but I am not able to access it and getting "This site can't be reached"

1. I want to know where I am making mistake?
2. What is use of $uri$is_args$args; in proxy_pass ?
3. Is rewrite needed ?

**Comment by buchdag** [comment]
Hi. I'm not sure you got it right, this is the repository for the [nginx-proxy](https://hub.docker.com/repository/docker/nginxproxy/nginx-proxy) Docker image, not for nginx.
